widgetastic_manageiq.py

```diff
@@ -1205,15 +1206,15 @@ class ItemsToolBarViewSelector(View):
         view_selector.selected
     """
     ROOT = './/div[contains(@class, "toolbar-pf-view-selector")]'
-    grid_button = Button(title='Grid View')
-    tile_button = Button(title='Tile View')
-    list_button = Button(title='List View')
+    btn_class = VersionPick({
```

Comment:
It can be done like with ManageIQTree ... btw I would avoid the btn_class here and just place the verpicked buttons on the class.